pkg/infra/main.go

```diff
@@ -223,6 +223,7 @@ func startGameLoop() {
 				logging.OutputLog(logging.Loss)
 
 				csvFile.Close()
+				fmt.Println(csvFile.Name())
```

Comment:
deliberately removed this since all have same name

Reply:
Hm not really (although that might seem a bit ambiguous https://pkg.go.dev/os#Create), i just added that for the sake of completeness in order to use the `runtest.sh`

Reply:
I named them all output.csv for post-processing